.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+|**Field**|**Value**|
+|:---|:---|
+|**_Submission Date_**| [DATE] |
+|**_Submitter_**| [USER] |
+|**_Originating Project/Creator_**| |
+|**_Base Branch_**| devel |
+|**_Short Description_**| |
+|**_Affected Component_**|  |
+|**_Affected Architectures(s)_**|  |
+|**_Related Issue(s)_**|  |
+|**_Has Unit Tests (y/n)_**|  |
+|**_Build Checked (y/n)_**|  |
```

Comment:
Build Checked sounds a little awkward to me. Would 'Changes Compile' possibly sound a bit better?